apps/expo-go/ios/Podfile.lock

```diff
@@ -2399,7 +2399,7 @@ SPEC CHECKSUMS:
   GoogleDataTransport: 54dee9d48d14580407f8f5fbf2f496e92437a2f2
   GoogleMaps: 032f676450ba0779bd8ce16840690915f84e57ac
   GoogleUtilities: 13e2c67ede716b8741c7989e26893d151b2b2084
-  hermes-engine: 118ab4d39602f2b05fd37ba7356343d2b5344b9c
+  hermes-engine: 1d242e1d7d6c63ee223040c5a0d72d59105ed811
```

Comment:
I think this hash is not correct. It seems to be unstable again... 😢 Probably better to just revert this change.